editor.planx.uk/src/pages/FlowEditor/lib/store/preview.ts

```diff
@@ -638,15 +638,18 @@ export const previewStore: StateCreator<
     }
   },
 
-  // TODO: Fix this!
-  // requestedFiles: () => {
-  //   const { computePassport } = get();
-  //   const currentRequestedFiles =
-  //     computePassport().data?.[PASSPORT_REQUESTED_FILES_KEY];
-  //   const emptyFileList = { required: [], recommended: [], optional: [] };
+  requestedFiles: () => {
+    // Importing PASSPORT_REQUESTED_FILES_KEY causes tests to fail - possible circular dependency issue?
+    // Repeating it here so find and replace still points to this hardcoded value
+    const PASSPORT_REQUESTED_FILES_KEY = "_requiredFiles";
```

Comment:
There might be an argument to move all `PASSPORT_*` consts to a single shared location if we come across a quirk like this again.